tests/unit/EzcapClient.spec.js

```diff
@@ -13,59 +16,114 @@ const didKeyDriver = didKey.driver();
 describe('ZcapClient', () => {
   describe('constructor', () => {
     it('should create an ZcapClient using a didDocument', async () => {
-      const {didDocument, keyPairs} = await didKeyDriver.generate();
+      didKeyDriver.use({
+        multibaseMultikeyHeader: 'z6Mk',
+        fromMultibase: Ed25519VerificationKey2020.from
+      });
+      const verificationKeyPair = await Ed25519VerificationKey2020.generate();
+      const {didDocument, keyPairs} =
+        await didKeyDriver.fromKeyPair({verificationKeyPair});
```

Comment:
Hmm, we might want to just remove this test. `keyPairs` would need to be updated for this to work properly. Someone might also copy this example and it isn't correct now. The `keyPairs` would need to be updated in some way to enable the use of signers.

Reply:
Right, that's what this issue is about: https://github.com/digitalbazaar/ezcap/issues/26

Reply:
In preparation for a fix somehow somewhere here, I think the appropriate thing is to demonstrate how to *properly* initialize the client.  And therefore proper keys should be passed in etc.  We now recognize that successfully calling the constructor does not produce a fully operational client instance.